ci/matlab.yml

```diff
@@ -0,0 +1,57 @@
+# This is a basic workflow to help you get started with MATLAB actions.
+# For more information, see https://github.com/matlab-actions/overview.
+# 
+# This workflow uses actions that are not certified by GitHub. 
+# They are provided by a third-party and are governed by 
+# separate terms of service, privacy policy, and support 
+# documentation. 
+
+name: MATLAB
+
+on:
+  push:
+    branches: [ $default-branch ]
+  pull_request:
+    branches: [ $default-branch ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      # Set up MATLAB on a GitHub-hosted runner
+      - name: Set up MATLAB
+        uses: matlab-actions/setup-matlab@v1
+
+      - name: Run tests and generate artifacts
+        uses: matlab-actions/run-tests@v1
+        with:
+          source-folder: CODE
+          code-coverage-cobertura: code-coverage/coverage.xml
+          test-results-junit: test-results/results.xml
+
+      - name: Upload code coverage report
+        uses: actions/upload-artifact@v2
+        with:
+          name: code-coverage
+          path: code-coverage/coverage.xml
+
+      - name: Upload test results
+        uses: actions/upload-artifact@v2
+        if: always()
+        with:
+          name: test-results
+          path: test-results/results.xml
+
+      # Uncomment this step to execute a MATLAB script, function, or statement
```

Comment:
The uncommented code will run a script, not a function or statement. Do we need to say the following instead?

# Uncomment this section to run a MATLAB script

Reply:
Rather than the "Uncomment this section..." header. Can we say something closer to what we say in the ci-config-example repo: 

"# As an alternative to run-tests, you can use run-command to execute a MATLAB script, function, or statement."

The "Uncomment this section" would lead me to believe I am meant to leave everything above and just uncomment this part.

Reply:
Updated both the comments